rust/src/tx_builder.rs

```diff
@@ -801,14 +806,14 @@ impl TransactionBuilder {
 
     /// Returns a copy of the current mint witness scripts in the builder
     pub fn get_mint_scripts(&self) -> Option<NativeScripts> {
```

Comment:
I'm not intended to break the existing interface but this could be `get_native_scripts` as well.